spec/app/ics-027-interchain-accounts/README.md

```diff
@@ -201,14 +208,15 @@ function onChanOpenAck(
   // port has already been validated
   // assert that version is "ics27-1"
   abortTransactionUnless(version === "ics27-1")
+  // state change to keep track of successfully registered interchain account
+  confirmInterchainAccountRegistration()
 }
 ```
 
 ```typescript
 function onChanOpenConfirm(
   portIdentifier: Identifier,
   channelIdentifier: Identifier) {
-  // accept channel confirmations, port has already been validated
 }
 ```
 
```

Comment:
In case channel is closed, access to the interchain account is lost if I am not wrong. If that's true, access to assets is lost, so I wonder if there is something we can do here.

Reply:
I'm thinking there are probably several ways to achieve reclaiming an account. 

A question that comes to mind: Do we want several channels to be active at one time, all of which can send instructions to the account? 

Also, if owner A needs to reclaim an account, what channel are they using to do that? In the case of a DAO the reclaiming process would need to be communicated over IBC.

Reply:
I think we should just maintain a single active channel at a time. Multiple channels at a time means there's no global ordering of messages